index.html

```diff
@@ -41,45 +41,45 @@ <h2>Skills</h2>
           <li>Audio Recording & Editing on Cubase</li>
         </ul>
       </section>
-      <section id="contact">
+      <section class="infobox" id="contact">
         <h2>Contact</h2>
         <ul>
           <li>E-Mail: dhs@schrader-wolff-media.de</li>
           <li>Website: www.beerserker.de</li>
         </ul>
       </section>
-      <section id="socialmedia">
+      <section class="infobox" id="socialmedia">
         <h2>Find me on</h2>
-        <div class="social">
+        <div class="infobox__social">
           <figure>
             <a href="https://github.com/DanSchrader">
-              <img class="logo" src="./images/394187_github_icon.png" alt=""
+              <img class="infobox__social__logo" src="./images/394187_github_icon.png" alt=""
```

Comment:
```suggestion
              <img class="infobox__logo" src="./images/394187_github_icon.png" alt=""
```

BEM rules only use one __